exporters/metric/datadog/example_test.go

```diff
@@ -67,7 +68,7 @@ func ExampleExporter() {
 		global.SetMeterProvider(pusher.MeterProvider())
 		meter := global.Meter("marwandist")
 		m := metric.Must(meter).NewInt64Histogram("myrecorder")
-		meter.RecordBatch(context.Background(), []attribute.KeyValue{attribute.Int("l", 1)},
+		meter.RecordBatch(context.Background(), []attribute.KeyValue{attribute.Int("l", 1), semconv.ServiceNameKey.String("ExampleExporter")},
```

Comment:
`service_name` is a resource attribute, so this should be handled by passing a `*Resource` to `controller.New()` via the [`WithResource()`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric@v0.23.0/controller/basic#WithResource) option.

Reply:
No problem

please review again 🙃